source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,150 @@
+
+If you are upgrading from the previous LTS version to 3.12 then you are getting access to these features:
+
+[3.6](https://blog.emberjs.com/2018/12/13/ember-3-6-released.html)
+* `router` service 
+* Support for native classes
+
+[3.8](https://blog.emberjs.com/2019/02/27/ember-3-8-released.html)
+* The array helper
+* Element Modifier Manager
+
+[3.10](https://blog.emberjs.com/2019/05/21/ember-3-10-released.html)
+* Nested angle bracket component invocation
+* Angle bracket invocation for built-in components
+* Support for native decorators
+
+[3.11](https://blog.emberjs.com/2019/07/15/ember-3-11-released.html)
+* Forwarding Element Modifiers with `...attributes`
+* The `{{fn}}` and `{{on}}` modifiers
+
+This release kicks off the 3.14 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.13
+
+Ember.js 3.13 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are five (5) new features, one (1) deprecation, and several bugfixes in this version.
+
+#### New Features (5)
+
+**Tracked Properties and Tracked Property Updates (1 of 5)**
+
+Tracked properties introduce a simpler and more ergonomic system for tracking state change in Ember applications. By taking advantage of new JavaScript features, tracked properties allow Ember to reduce its API surface area while producing code that is both more intuitive and less error-prone.
+
+During the Ember Octane preview period we encountered some issues with the current design for Tracked Properties that was proposed and accepted in [RFC 410](https://github.com/emberjs/rfcs/blob/master/text/0410-tracked-properties.md). The primary issues were specifically around interop between tracked properties, computed properties, and autotracking, with a few extra issues and inconsistencies surrounding these.
```

Comment:
I believe there were also, and equally seriously, issues around interop with sync observers—and the fixes made with them have performance implications that people should know about here. cc. @pzuraq.

Reply:
Yes, observer heavy applications should definitely keep an eye out for performance regressions with this release, and consider switching on the async-observers optional feature in order to mitigate if they find issues. We should note that here for sure.

Reply:
Should we just add async observers as a separate top-level items?